x-pack/plugin/inference/src/main/java/module-info.java

```diff
@@ -22,10 +22,5 @@
     exports org.elasticsearch.xpack.inference.registry;
     exports org.elasticsearch.xpack.inference.rest;
     exports org.elasticsearch.xpack.inference.services;
-    exports org.elasticsearch.xpack.inference.external.http.sender;
```

Comment:
@davidkyle do we need these? I had added them a while ago but now I get all kinds of warnings saying that various classes aren't exported. If I remove them they go away 🤷‍♂️ . I don't think we'd want to export most of this stuff out of the module anyway.